.buildbaselog

```diff
@@ -11,6 +11,9 @@
 *   Jul 15 2021 <stone.zhang@broadcom.com>
 -   refresh base image
 
+*   Jan 14 2024 <yan-yw.wang@broadcom.com>
```

Comment:
2024->2025

Reply:
Please also make sure the texts in this file are ordered correctly.  The newest entry should be on top IMO.

Reply:
fixed.